examples/tokio-fetch/src/lib.rs

```diff
@@ -0,0 +1,106 @@
+use neon::prelude::*;
+use once_cell::sync::OnceCell;
+use serde::Deserialize;
+use tokio::runtime::Runtime;
+
+#[derive(Deserialize)]
+struct NodeRelease {
+    version: String,
+    date: String,
+}
+
+// Return a global tokio runtime or create one if it doesn't exist.
+// Throws a JavaScript exception if the `Runtime` fails to create.
+fn runtime<'a, C: Context<'a>>(cx: &mut C) -> NeonResult<&'static Runtime> {
+    static RUNTIME: OnceCell<Runtime> = OnceCell::new();
+
+    RUNTIME.get_or_try_init(|| Runtime::new().or_else(|err| cx.throw_error(err.to_string())))
+}
+
+// Get the verson of the currently running node process from `process.version`
```

Comment:
Add a link to the `process.version` docs ( https://nodejs.org/api/process.html#processversion ) to this comment?